extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -15,6 +15,7 @@
  */
 
 import {IntersectionObserver} from '../../../src/intersection-observer';
+import {isPositionFixed} from '../../amp-ad/0.1/amp-ad-3p-impl';
```

Comment:
This should fail a presubmit :)

Please factor into a common dependency in `src/`